src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorDocumentMappingService.cs

```diff
@@ -105,16 +104,16 @@ public override VS.TextEdit[] GetProjectedDocumentEdits(RazorCodeDocument codeDo
                     // so we can ignore all but the last line. This assert ensures that is true, just in case something changes in Roslyn
                     Debug.Assert(lastNewLine == 0 || edit.NewText.Substring(0, lastNewLine - 1).All(c => c == '\r' || c == '\n'), "We are throwing away part of an edit that has more than just empty lines!");
 
-                    var proposedRange = new Omni.Range(range.End.Line, 0, range.End.Line, range.End.Character);
+                    var proposedRange = new VS.Range { Start = new VS.Position(range.End.Line, 0), End = new VS.Position(range.End.Line, range.End.Character) };
```

Comment:
Ah still need VS because of `System.Range` right?

Reply:
Ah yeah good catch.